extensions/tournament.py

```diff
@@ -0,0 +1,113 @@
+from datetime import datetime, timedelta
+
+import discord
+from discord.ext import commands
+
+from db import db_session
+from extensions.util import remove_reaction, create_role_and_channels
+from models.tournament import Tournament
+
+
+class Tournaments(commands.Cog, name="Tournaments"):
+
+    def __init__(self, bot):
+        self.bot: discord.Client = bot
+
+    @commands.command()
+    async def tournament(self, ctx, role: discord.Role, team_size: int, team_count: int, period: int = 15):
```

Comment:
Should we check if {team_count} is an even number or better if it fits the {team_count} structure?

Reply:
Some validation should be added, for sure. Currently you could simply put `team_size` at 0 and nobody can create a team 😂 
An even number would be easier if all tournaments will only consist of 1v1 matches, however I think it would be interesting to also support automatic group creation with a round-robin match schedule or something similar.

However, for a first test it might be easier to force an even number.

Reply:
Sounds good to me